Sources/IO/Misc/PDBReader/example/index.js

```diff
@@ -26,6 +26,7 @@ const sphereActor = vtkActor.newInstance();
 const stickActor = vtkActor.newInstance();
 
 filter.setInputConnection(reader.getOutputPort());
+filter.setHideElement('H');
```

Comment:
The array macro is doing exactly that. So you just need to manage the `size` argument since you don't know how many you will have.